pkg/assembler/backends/testing/cve.go

```diff
@@ -70,6 +70,28 @@ type cveIDNode struct {
 func (n *cveIDNode) getID() uint32 { return n.id }
 func (n *cveNode) getID() uint32   { return n.id }
 
+func (n *cveNode) neighbors() []uint32 {
+	out := make([]uint32, 0, len(n.cveIDs))
+	for _, v := range n.cveIDs {
+		out = append(out, v.id)
+	}
+	return out
+}
+
+func (n *cveIDNode) neighbors() []uint32 {
+	// TODO: DO NOT MERGE: add backedge
```

Comment:
> DO NOT MERGE

oops =)

Reply:
Yeah, I have it to know to integrate the new PRs after rebasing :)